gcloud/monitoring/_helpers.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helper functions for shared behavior."""
+
+
+def _format_timestamp(timestamp):
```

Comment:
This function is closest to the function `_datetime_to_rfc3339` under `gcloud._helpers`, except that it converts zone aware datetime objects to UTC.

Would it make sense to extend that to take in an extra boolean argument (`is_zone_aware`) and do zone conversion when requested?

**Note**: This function was earlier part of `gcloud.monitoring.query`, but was moved here since the groups API also needs to use it.